.github/workflows/ci.yml

```diff
@@ -9,6 +9,13 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
+  APT_DEPS: >-
+    libglib2.0-dev
+    gcc
+    gettext
+    libpam0g-dev
+
 jobs:
   analyze:
```

Comment:
as you have some parts specific to Go, this is specific to flutter/dart and should be marked as such?

Reply:
Yes. In my mind it makes sense to mark `flutter-tests`, `flutter-analyze`, `flutter-coverage`, `flutter-format` & `flutter-mocks`?